crates/sui-replay-frontend/src/main.rs

```diff
@@ -0,0 +1,12 @@
167,143,157,217,17,42,226,13,38,190,255,55,115,142,219,33,199,188,101,182,179,95,122,39,236,15,90,110,126,97,52,180,111,35,13,86,207,88,97,195,40,106,114,79,207,50,228,214,224,11,154,197,114,15,210,17,77,159,149,220,142,77,228,112,104,24,188,241,240,94,107,226,156,15,159,187,111,81,45,182,249,236,216,164,174,137,201,102,134,46,208,235,30,69,125,216,108,72,244,103,71,45,33,223,153,130,157,2,14,236,253,218,202,232,195,159,129,225,29,186,69,196,20,100,64,104,118,189,144,220,199,220,64,211,15,191,123,63,3,229,245,182,49,77,143,148,28]},{\"Pure\":[1,0,0,0,0,0,0,0]},{\"Pure\":[1,0,0,0,0,0,0,0]},{\"Pure\":[1,0,0,0,0,0,0,0]},{\"Object\":{\"SharedObject\":{\"id\":\"0xf28e4e40a96948a19805fe76a4da373a262daac47588bac78d73033bce7b5a38\",\"initial_shared_version\":20119534,\"mutable\":true}}},{\"Object\":{\"SharedObject\":{\"id\":\"0x6e31a26e9f903db7b18919d4bdd7b7634ee890bc631bf116e6f9f24ab16f1315\",\"initial_shared_version\":20120865,\"mutable\":true}}},{\"Object\":{\"SharedObject\":{\"id\":\"0x29e37978cb1c9501bda5d7c105f24f0058bc1668637e307fbc290dba48cb918d\",\"initial_shared_version\":20120875,\"mutable\":true}}}],\"commands\":[{\"MoveCall\":{\"package\":\"0x5306f64e312b581766351c07af79c72fcb1cd25147157fdc2f8ad76de9a3fb6a\",\"module\":\"vaa\",\"function\":\"parse_and_verify\",\"type_arguments\":[],\"arguments\":[{\"Input\":0},{\"Input\":1},{\"Input\":2}]}},{\"MoveCall\":{\"package\":\"0x04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"pyth\",\"function\":\"create_authenticated_price_infos_using_accumulator\",\"type_arguments\":[],\"arguments\":[{\"Input\":3},{\"Input\":4},{\"NestedResult\":[0,0]},{\"Input\":2}]}},{\"SplitCoins\":[\"GasCoin\",[{\"Input\":5},{\"Input\":6},{\"Input\":7}]]},{\"MoveCall\":{\"package\":\"0x04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"pyth\",\"function\":\"update_single_price_feed\",\"type_arguments\":[],\"arguments\":[{\"Input\":3},{\"NestedResult\":[1,0]},{\"Input\":8},{\"NestedResult\":[2,0]},{\"Input\":2}]}},{\"MoveCall\":{\"package\":\"0x04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"pyth\",\"function\":\"update_single_price_feed\",\"type_arguments\":[],\"arguments\":[{\"Input\":3},{\"NestedResult\":[3,0]},{\"Input\":9},{\"NestedResult\":[2,1]},{\"Input\":2}]}},{\"MoveCall\":{\"package\":\"0x04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"pyth\",\"function\":\"update_single_price_feed\",\"type_arguments\":[],\"arguments\":[{\"Input\":3},{\"NestedResult\":[4,0]},{\"Input\":10},{\"NestedResult\":[2,2]},{\"Input\":2}]}},{\"MoveCall\":{\"package\":\"0x04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"hot_potato_vector\",\"function\":\"destroy\",\"type_arguments\":[{\"struct\":{\"address\":\"04e20ddf36af412a4096f9014f4a565af9e812db9a05cc40254846cf6ed0ad91\",\"module\":\"price_info\",\"name\":\"PriceInfo\",\"type_args\":[]}}],\"arguments\":[{\"NestedResult\":[5,0]}]}}]}}}";
```

Comment:
Just a thought, have this in a separate file and read it instead? If you need to change something later, you don't need to recompile main.